modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheContextInfo.java

```diff
@@ -69,7 +89,7 @@ public GridCacheContextInfo(GridCacheContext<K, V> cctx, boolean clientCache) {
      * @param cacheDesc Cache descriptor.
      */
     public GridCacheContextInfo(DynamicCacheDescriptor cacheDesc) {
-        config = cacheDesc.cacheConfiguration();
+        config = new CacheConfiguration(cacheDesc.cacheConfiguration());
```

Comment:
Let's use `new CacheConfiguration<>` to remove compiler warning

Reply:
This doesn't help, cacheDesc.config() returns not parameterized CacheConfiguration.